app/views/directives/edit-environment-variables.html

```diff
@@ -32,6 +34,21 @@
       add-row-with-selectors-link="Add Environment Variable Using a Config Map or Secret"
       show-header>
     </key-value-editor>
+
+    <h4>
+      Environment From
+      <span class="pficon pficon-info"
```

Comment:
We usually use a `?`(`pficon-help`) for these tooltips, for instance:

![openshift web console 2017-09-26 09-25-53](https://user-images.githubusercontent.com/1167259/30862644-c40afce8-a29c-11e7-84cb-7542f7f28df9.png)

@beanh66 Any objection to switching for consistency?